### PR TITLE
feat: new verifier operations doc

### PR DESCRIPTION
**Pull request by ffe9f8**
New verifier operations doc.

Preview: https://axelar-docs-git-marty-verifier-operations-axelar-network.vercel.app/validator/amplifier/verifier-operations

### CODE REVIEW

File: src/pages/validator/amplifier/verifier-operations.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Rewards
+
+There is one [rewards contract](https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/rewards/src/msg.rs) for all chain integrations. Rewards are paid out in $AXL.
+
+### How rewards are calculated
+
+Every action earns rewards within `n` epochs. `n` more epochs will be paid out based on what has already been paid. There is a single global parameter for each epoch called `rewards_per_epoch`, in `Uint128`. Later, there will be one per pool. 
+
+Verifiers must hit a [PLACEHOLDER] threshold for payout.
```
Comment (ffe9f8) [inline src/pages/validator/amplifier/verifier-operations.mdx:27]: What's the threshold?

File: src/pages/validator/amplifier/verifier-operations.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+The Axelar Virtual Machine (AVM) and Amplifier are currently under active development, so these instructions are likely to change. Please check back frequently for updates.
+</Callout>
+
+## Approvals
+
+### Approve a verifier or verifier set
+
+Each verifier in a verifier set must use `ampd` to bond and register chain support themselves. On testnet and mainnet, community governance will call the following to authorize them:
+
+```bash
```
Comment (StephenFluin) [inline src/pages/validator/amplifier/verifier-operations.mdx:15]: This is how you do it if you have permissions, but we need to teach how to do it via governance proposal (AKA how to submit the actual proposal)

File: src/pages/validator/amplifier/verifier-operations.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+axelard tx wasm execute [service registry address] '{"authorize_workers":{"workers":[list of worker addresses], "service_name":[service-name]}}' --from key --gas auto --gas-adjustment 2
+```
+
+## Rewards
+
+There is one [rewards contract](https://github.com/axelarnetwork/axelar-amplifier/blob/main/doc/src/contracts/rewards.md) for all chain integrations. Rewards are paid out in $AXL.
```
Comment (StephenFluin) [inline src/pages/validator/amplifier/verifier-operations.mdx:21]: Remove `$`, the token symbol is AXL.

Comment (StephenFluin) [reply src/pages/validator/amplifier/verifier-operations.mdx:21]:
```suggestion
There is one [rewards contract](https://github.com/axelarnetwork/axelar-amplifier/blob/main/doc/src/contracts/rewards.md) for all chain integrations. Rewards are paid out in AXL.
```

File: src/pages/validator/amplifier/verifier-operations.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+### How rewards are calculated
+
+Every action earns rewards within `n` epochs. `n` more epochs will be paid out based on what has already been paid. There is a single global parameter for each epoch called `rewards_per_epoch`, in `Uint128`. Later, there will be one per pool.
```
Comment (StephenFluin) [inline src/pages/validator/amplifier/verifier-operations.mdx:25]: > Every action earns rewards within `n` epochs. `n` more epochs will be paid out based on what has already been paid. 
This is really unclear to me, maybe we can find a more specific way to explain it?

We should leave out the promise and just talk about how things work today, or pull all of these little notes out into their own section.

File: src/pages/validator/amplifier/verifier-operations.mdx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+## Rewards
+
+There is one [rewards contract](https://github.com/axelarnetwork/axelar-amplifier/blob/main/doc/src/contracts/rewards.md) for all chain integrations. Rewards are paid out in $AXL.
+
+### How rewards are calculated
+
+Every action earns rewards within `n` epochs. `n` more epochs will be paid out based on what has already been paid. There is a single global parameter for each epoch called `rewards_per_epoch`, in `Uint128`. Later, there will be one per pool. 
+
+A [threshold of weighted votes](https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/voting-verifier/src/msg.rs#L25) is required for voting to be considered complete for a particular message. Verifiers must hit that threshold for payout.
```
Comment (StephenFluin) [inline src/pages/validator/amplifier/verifier-operations.mdx:27]: I'm confused. The voting threshold is independent from payouts. Maybe we mean a vote must pass threshold for the verifiers to qualify for `RecordParticpiation`? I'm not sure that's true?

File: src/pages/validator/amplifier/verifier-operations.mdx [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+### Check a rewards pool
+
+To check a rewards pool:
+
+```bash
+axelard tx wasm execute $CONTRACT_ADDRESS '' --from validator --gas auto --gas-adjustment 2
+axelard q wasm contract-state smart [rewards contract address] '{"rewards_pool":{"pool_id":{"chain_name":[chain-name],"contract":[address of voting verifier or multisig]}}}'
```
Comment (StephenFluin) [inline src/pages/validator/amplifier/verifier-operations.mdx:35]: What do you think about moving these commands onto multiple lines like we did in the integrator guide? We should also consider making an explicit variable block like we did too.

File: src/pages/validator/amplifier/verifier-operations.mdx [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+`contract` is the address of the voting verifier for the chain when querying for voting rewards, but the global multisig (not multisig prover) address when querying for signing.
+
+### Add rewards to a pool or chain
+
+To add rewards to a pool:
+
+```bash
+axelard tx wasm execute [rewards contract address] '{"add_rewards":{"pool_id":{"chain_name":[chain-name],"contract":[address of voting verifier or multisig]}}}' --amount [amount to add] --from [key] --gas auto --gas-adjustment 2
```
Comment (StephenFluin) [inline src/pages/validator/amplifier/verifier-operations.mdx:45]: Have you run all of these commands? We'll likely want to include other params like --node until we're out of devnet.

**Review by StephenFluin — CHANGES_REQUESTED** [review]

File: src/pages/validator/amplifier/verifier-operations.mdx [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+</Callout>
+
+## Approvals
+
+### Approve a verifier or verifier set
+
+Each verifier in a verifier set must use `ampd` to bond and register chain support themselves. On testnet and mainnet, community governance will call the following to authorize them:
+
+```bash
+axelard tx wasm execute [service registry address] '{"authorize_workers":{"workers":[list of worker addresses], "service_name":[service-name]}}' --from key --gas auto --gas-adjustment 2
```
Comment (StephenFluin) [inline src/pages/validator/amplifier/verifier-operations.mdx:16]: make sure these match 0.4.0 with worker -> verifier

File: src/pages/validator/amplifier/verifier-operations.mdx (hunk 4 above)
Comment (ffe9f8) [reply src/pages/validator/amplifier/verifier-operations.mdx:25]: Leave out "Later, there will be one per pool."

File: src/pages/validator/amplifier/verifier-operations.mdx (hunk 5 above)
Comment (StephenFluin) [reply src/pages/validator/amplifier/verifier-operations.mdx:27]:
```suggestion
A [threshold of weighted votes](https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/voting-verifier/src/msg.rs#L25) is required for voting to be considered complete for a particular message.
```

File: src/pages/validator/amplifier/verifier-operations.mdx (hunk 7 above)
Comment (StephenFluin) [reply src/pages/validator/amplifier/verifier-operations.mdx:45]: You should be able to use this https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json and add rewards to a pool without any permissions.

**Comment by StephenFluin** [comment]
@ffe9f8 is splitting into two pages

File: src/pages/validator/amplifier/verifier-operations.mdx (hunk 2 above)
Comment (ffe9f8) [reply src/pages/validator/amplifier/verifier-operations.mdx:15]: Refer to https://docs.axelar.dev/learn/cosmwasm-governance

**Pull request closed without merging**